src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int CAPACITY = 16;
```

Comment:
```suggestion
    private static final int INITIAL_CAPACITY = 16;
```
your capacity changes after resize, so it won't always be 16